### PR TITLE
CT-2963 Page counts data & f/e

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Add 3 Page counts for total, exempt and dispatched

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`

### Screenshots
Before:
image

After:

image

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2963

### Deployment
some new data fields involved

### Manual testing instructions
Go to any offender SARS case and add data requests, and total pages, exempt pages, dispatched pages. Then check the status box on the Show page.

### CODE REVIEW

File: app/controllers/concerns/offender_sar_cases_params.rb [hunk 1]
```diff
@@ -26,11 +29,14 @@ def create_offender_sar_params
       :third_party_name,
       :third_party_relationship,
       :third_party_company_name,
+      :date_responded_dd, :date_responded_mm, :date_responded_yyyy,
+      :date_of_birth_dd, :date_of_birth_mm, :date_of_birth_yyyy,
```
Comment (willmcb) [inline app/controllers/concerns/offender_sar_cases_params.rb:33]: Were these deleted and re-added? I thought it these fields were already in the app?

File: spec/factories/case/offender_sars.rb [hunk 2]
```diff
@@ -35,6 +35,9 @@
     recipient                       { 'subject_recipient' }
     third_party                     { false }
     flag_as_high_profile            { false }
+    num_exempt_pages                { '200' }
```
Comment (willmcb) [inline spec/factories/case/offender_sars.rb:38]: Think this needs to be `number_*` rather than `num_*` 👍

**Review by willmcb — CHANGES_REQUESTED** [review]
Other than the comment about `number_*` to `num_*` this looks good - I think the tests cover different aspects to my upcoming PR so add value and protect against unintended side effects. 👍

File: app/controllers/concerns/offender_sar_cases_params.rb (hunk 1 above)
Comment (njpreddy) [reply app/controllers/concerns/offender_sar_cases_params.rb:33]: I think I deleted them by accident and re-added

File: spec/factories/case/offender_sars.rb (hunk 2 above)
Comment (njpreddy) [reply spec/factories/case/offender_sars.rb]: These attributes have been removed from the factory.

**Review by willmcb — APPROVED** [review]
👍

**Pull request merged**